src/index.html

```diff
@@ -0,0 +1,217 @@
+                                <input type="hidden" name="idmoovie" value="" class="moovieId">
+                                  <p class="likes-number"></p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="moovie-item-options">
+                        <button type="button" class="btnComment">Comments</button>
+                        <button type="button" class="btn-reserve">Reservation</button>
+                    </div>
+                </div>
+                <div class="moovie-item">
+                    <div class="moovie-item-image-container">
+                        <div class="moovie-item-picture">
+                        </div>
+                        <div class="moovie-item-image-description">
+                            <h4 class="item-name">Name</h4>
+                            <div class="moovie-item-likes">
+                                <p class="heart"><i class="fa-regular fa-heart" id="likeIcon"></i></p>
+                                <input type="hidden" name="idmoovie" value="" class="moovieId">
+                                  <p class="likes-number"></p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="moovie-item-options">
+                        <button type="button" class="btnComment">Comments</button>
+                        <button type="button" class="btn-reserve">Reservation</button>
+                    </div>
+                </div>
+                <div class="moovie-item">
+                    <div class="moovie-item-image-container">
+                        <div class="moovie-item-picture">
+                        </div>
+                        <div class="moovie-item-image-description">
+                            <h4 class="item-name">Name</h4>
+                            <div class="moovie-item-likes">
+                                <p class="heart"><i class="fa-regular fa-heart" id="likeIcon"></i></p>
+                                <input type="hidden" name="idmoovie" value="" class="moovieId">
+                                  <p class="likes-number"></p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="moovie-item-options">
+                        <button type="button" class="btnComment">Comments</button>
+                        <button type="button" class="btn-reserve">Reservation</button>
+                    </div>
+                </div>
+                
+            </div>
+        </section>
+        <div class="seemoreContainer">
+            <button type="button"  id="btnSee"><i class="fa-solid fa-eye-slash"></i> See Less</button>
+        </div>
+    </main>
+    
+    <section class="popup-container hide">
+    </section>
+    
+    <footer>
+            <p>Created by Microverse students under MIT Licence</p>
+    </footer>
```

Comment:
The footer should be separated from the main body by a border as in the provided wireframe design. You don't need to go crazy with the styling but your design should follow the provided design.

| Your design  | Provided design |
| ------------- | ------------- |
| ![image](https://github.com/Graciano1997/js_moduleII_capstone/assets/33268927/e7c59d34-312d-4df9-bcb6-2cd0fca641ce) | ![image](https://github.com/Graciano1997/js_moduleII_capstone/assets/33268927/5da537d7-db62-4a73-aa38-d1133edcd716) |